pandas/core/generic.py

```diff
@@ -2909,6 +2909,9 @@ def _drop_axis(self, labels, axis, level=None, errors='raise'):
             else:
                 indexer = ~axis.isin(labels)
 
+            if all(indexer) and errors == 'raise':
```

Comment:
I believe `indexer` is a boolean numpy array, right?  In that case I think you'll want to use `indexer.all()` instead of `all(indexer)`.  The latter will be slower if `indexer` is at all sizable, since builtin `all` requires conversion to python objects, whereas `.all()` does everything within numpy:

```python
In [2]: a = np.repeat([True], 1000)

In [3]: %timeit a.all()
1.81 µs ± 87.6 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [4]: %timeit all(a)
17.2 µs ± 609 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)
```